lib/blocking-io.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function () {
+  if (process.stdout._handle && typeof process.stdout._handle.setBlocking === 'function') {
+    process.stdout._handle.setBlocking(true)
+    process.stderr._handle.setBlocking(true)
+  }
```

Comment:
I suppose this doesn't ever manifest as an issue before 0.11.2 then, but I haven't tested it.